File: tests/fixtures/v2-process-flow/positive/process-flow.js
function direct(req, child_process) {
  return child_process.exec(req.query.direct)
}

function propagated(req, child_process) {
  const command = req.query.propagated
  const alias = command
  return child_process.exec(alias)
}

function separated(req, child_process) {
  return child_process.execFile('tool', [req.query.argument])
}

File: tests/fixtures/v2-process-flow/positive/ProcessFlow.cs
class ProcessFlow
{
    void Direct(dynamic Request)
    {
        Process.Start(Request.Query["direct"]);
    }

    void Propagated(dynamic Request)
    {
        var command = Request.Query["propagated"];
        var forwardedValue = command;
        Process.Start(forwardedValue);
    }

    void Separated(dynamic Request, dynamic startInfo)
    {
        startInfo.ArgumentList.Add(Request.Query["argument"]);
    }
}
